### DISCUSSION TITLE
USE THE PUSH BUTTON TO CONTROL LED IN EXAMPLE 15

### DISCUSSION THREAD

**Discussion by Le-Duc0069** [HomeSpan/HomeSpan#586] [General]
hello homespan, i have this problem, in example 15 i want to change to 1 button to turn the light on and off and try to respond with hamekit app, no need to adjust the brightness what do i need to fix, if possible Can you help me to fix it, THANK YOU SO MUCH

**Comment by HomeSpan** [comment]
Rather than use the dimming logic in Example 15 to control the LED, use the logic in Example 5 which just contains a simple LED.  Alternatively, start with Example 5 and copy over the Button logic from Example 15.
